darwin/DarwinProcess.h

```diff
@@ -20,6 +20,7 @@ typedef struct DarwinProcess_ {
    uint64_t stime;
    bool taskAccess;
    bool translated;
+   bool isThread;
```

Comment:
Should be handled via global `Process::isUSerlandThread` field instead of introducing this new one.